aws/table_aws_s3_bucket.go

```diff
@@ -23,9 +23,15 @@ func tableAwsS3Bucket(_ context.Context) *plugin.Table {
 			Hydrate: listS3Buckets,
 			Tags:    map[string]string{"service": "s3", "action": "ListBucket"},
 		},
+
 		// Note: No Get for S3 buckets, since it must list all the buckets
 		// anyway just to get the creation_date which is only available via the
 		// list call.
+
+		// Using IgnoreConfig in the Hydrate function has been observed to increase query execution time significantly.
+		// Therefore, we have opted not to use IgnoreConfig in Hydrate calls to maintain optimal performance.
+		// Example: For the query "select * from aws_s3_bucket where region = 'us-east-2'",
+		// using IgnoreConfig results in a slower execution time of 87.2 seconds, while handling errors manually reduces the time to 25.0 seconds.
```

Comment:
That's a very important notice, thanks!

Are there any solutions to improve this being considered?